src/util/sort-fn.js

```diff
@@ -12,5 +12,5 @@ module.exports = (a, b) => {
     }
     return String(bEle).localeCompare(String(aEle));
   }
-  return -1;
+  return 0;
```

Comment:
I'm now questioning to replace this with an assert again. It should never happen so yes... TODO

Reply:
Could also make it an explicit throw

Reply:
Done